scripts/header_parser.py

```diff
@@ -666,17 +701,39 @@ def main():
     parser.add_argument('-n', dest='namespace', metavar='namespace',
                         help='path to project namespace file (default ./NAMESPACE)',
                         default='./NAMESPACE')
+    parser.add_argument('-l', dest='log_file', metavar='logfile',
```

Comment:
Consider adding verbosity flag which controls verbosity level, i.e.
 - no flag: warning level
 - single flag (`-v`) : info level
 - double flag (`-vv`) : debug level

Reply:
Added along with `-q` and `-qq` to set error and critical levels, respectively